### ISSUE TITLE
Feedback Score

### ISSUE THREAD

**Issue by tjoye20** [haratzj/colmar#1]
### Criteria 1: HTML Structure  
* _Score Level:_ 4 - Exceeds Expectations
 
### Criteria 2: Visual Layout for Both Desktop and Mobile Sizes
* _Score Level:_ 3 - Meets Expectations
* _Comment(s):_ Your nav logos are broken in both the desktop and mobile views. Otherwise, really good job on this project. 
 
### Criteria 3: Responsive Design (Media Queries, Responsive Units, etc.)
* _Score Level:_ 4 - Exceeds Expectations
 
### Criteria 4: Visual Design and Accessibility (Color Palette, Typography, Transitions, etc.)
* _Score Level:_ 4 - Exceeds Expectations
 
### Overall Score: 15/16
Great job on this project, Seunghoon. It's clear you have a strong understanding of HTML, CSS, and responsive design. Make sure to look into fixing your broken image icons. Otherwise, keep up the good work!

**Comment by jshji252** [comment]
Hi, thanks for the review. However, the broken nav icons in Criteria 2, I am not sure which one you are talking about. can you please show me the broken logos maybe through a screenshot? thanks.

**Comment by jshji252** [comment]
I dont think we can attach photos here, maybe you can email me or you can message me on Codecademy if it is possible, or any other possible ways you can think of is fine haha. I want to see the problem so that i can fix it, because on my devices (macbook and iphone) it looks okay and i tried different browsers as well. thank you. if you can email me the screenshot, you can reach me here: jasonseunghoonji@gmail.com. Thank you again.

**Comment by tjoye20** [comment]
Line 60 of your index.html, `<img id="otdesktop"src="resources/images/information-orientation.jpg">`, you see how the src path is should be like this `./resources/images`. It's just missing that `./`.  Look throughout your index.html, a lot of the paths are missing that.
